docs/docs/data-structures/tree/nodes.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Tree Nodes
+sidebar_position: 1
+---
+
+Information is stored on a [SharedTree](./index.mdx) in the form of tree nodes.
```

Comment:
Does information mean the same as the data in the shared tree? I guess I am not sure what's the difference is between this line and the first line in `node_types.mdx`. Do they mean the same thing?

Reply:
They do, I changed this so that it's not duplicated.

Reply:
I think it's okay for it to be duplicated if it makes it less confusing.